index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
+	becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
+	<em class="rfc2119">should</em> solicit review from some or all of the following
+	stakeholders: 
+
+	<ul>
+		<li>The Working Group responsible for the specification</li>
+		<li>The horizontal review groups (see 6.2.3.2)</li>
+		<li>The Working Groups and organisations documented in the Working Group's charter</li>
+		<li>The general public</li>
+		<li>Other relevant standards organizations</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of wide review before
+	approving transitions. Appropriate evidence may include: 
+
+	<ul>
+		<li>Documented requests for wide review</li>
+		<li>Documented discussion and resolution of issues raised during wide review</li>
+	</ul>
+
+<h5 id="horizontal-review">
+Horizontal Review</h5>
+
+	The objective of <dfn id=dfn-horizontal-review export>horizontal review</dfn> is
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
```

Comment:
On what document is HR being solicited if the proposal hasn't yet made it onto the Rec Track? As I understand it a FPWD can be on the Rec Track. There may be no document to review before FPWD, though surely there would have been an Editor's Draft of some sort. Is this saying that before FPWD publication HR must be solicited on an ED?

Perhaps it would be more practical to have some sort of "intend to begin Rec work" notification either as part of a Charter process or, if the WG's Charter is liberal enough to permit new Rec track docs to be started by agreement of the WG, at some prior stage before FPWD.

Could HR be solicited on an Explainer?

Reply:
I think it is good practice to solicit HR before entering FPWD/the REC track (on the ED, or the CG Draft, or whatever precedes the FPWD), must solicit before seems to be the wrong way to go about it. It's only a requirement to solicit the feedback, not to have addressed it yet, so a mail 5 minutes before publishing would qualify, despite not being meaningfully different from a mail 5 minutes after, though that one wouldn't qualify. 

I think the requirement to send a mail to public-review-announce@w3.org is adequate as far as MUSTs are concerned, and does guarantee that FPWD will be the latest possible point to start soliciting HR. I'd also support add a should for soliciting review earlier, as it's still a good practice, but I would not make in blocking.

If we making HR solicitation before FPWD blocking, I expect that the effect may in many cases end up being the opposite of what we intend: instead of pushing groups to anticipate and start HR earlier than they planed to, it may delay entering the REC track, and along with it the start of HR.

The REC track is where the process begins to apply, and from that point on, we can apply the fairness guarantees that it provides, but not before, so delaying entrance on the REC track may cause the opposite of the intended effect: instead of forcing people to seek HR early, it could delay the point at which they enter the formal process, causing them to seek HR later.